egs/ami/s5/local/nnet3/run_lstm.sh

```diff
@@ -14,9 +14,29 @@ mic=ihm
 use_sat_alignments=true
 affix=
 speed_perturb=true
-splice_indexes="-2,-1,0,1,2 0"
 common_egs_dir=
 
+# lstm options
+splice_indexes="-2,-1,0,1,2"
+num_lstm_layers=1
+cell_dim=1024
+hidden_dim=1024
+recurrent_projection_dim=256
+non_recurrent_projection_dim=256
+truncated_sequence_width=20
+context_sensitive_chunk_width=20
```

Comment:
I am borrowing the terminology from Kai Chen, Zhi-Jie Yan, Qiang Huo "Training Deep Bidirectional LSTM Acoustic Model for LVCSR by a Context-Sensitive-Chunk BPTT Approach", Interspeech 2015

When users want to implement/use a feature based on paper, it would be easier to find if we use the same term as the paper to describe the option.

 Also left-context in the context of RNNs would not mean the same as left-context in FF-DNNs so I am bit wary of using the same name.